src/pages/home/report/ReportActionCompose.js

```diff
@@ -379,10 +385,10 @@ class ReportActionCompose extends React.Component {
      * @param {Boolean} shouldDebounceSaveComment
      */
     updateComment(newComment, shouldDebounceSaveComment) {
-        this.setState({
-            isCommentEmpty: !!newComment.match(/^(\s|`)*$/),
-            value: newComment,
-        });
```

Comment:
I know keeping `value` and `selection` in state is basically causing a re-render for every keypress, but ultimately, controlled components (state) should work as fast as uncontrolled with the new Fabric renderer

Reply:
Hey @kidroca thanks for the review. I actually just stashed some work and wasn't done yet, but thanks for having a look anyways already! I am not settling yet with this whole imperative approach with `setTextAndSelection`, but still investigating. I think i am actually gonna revert back to using `value` and `selection` but just in the "most child" composer component (for performance). 

However, i still wanted to compare performance between these two approaches, once with fabric and once without 😊  Will let you know once ready and re-tag you for review (all your other suggestions seem super reasonable, thx!)

Reply:
yes and no. This state was part of the `ReportActionCompose`. I moved the `value` state to the composer component. Thereby only a small component needs to rerender instead of the `ReportActionCompose`.

I wouldn't leave that `value` state in `ReportActionCompose` with fabric either